src/Venturecraft/Revisionable/RevisionableTrait.php

```diff
@@ -244,21 +426,16 @@ protected function dbInsert(array $revisions)
     }
 
     /**
-     * Attempt to find the user id of the currently logged in user
-     * Supports Cartalyst Sentry/Sentinel based authentication, as well as stock Auth
-     **/
+     * Attempt to find the user id of the currently logged in user.
+     *
+     * @return string|null
+     */
     private function getUserId()
     {
-        try {
-            if (class_exists($class = '\Cartalyst\Sentry\Facades\Laravel\Sentry')
-                || class_exists($class = '\Cartalyst\Sentinel\Laravel\Facades\Sentinel')
-            ) {
-                return ($class::check()) ? $class::getUser()->id : null;
-            } elseif (\Auth::check()) {
-                return \Auth::user()->getAuthIdentifier();
-            }
-        } catch (\Exception $e) {
-            return null;
+        $jwtAuth = App::make('Tymon\JWTAuth\JWTAuth');
```

Comment:
@artstorm yea sounds good, I'm pretty sure the new acl functionality uses the request methods so it would be good to use that for consistency.